x-pack/plugins/security_solution/public/common/components/filter_group/index.tsx

```diff
@@ -110,6 +111,9 @@ const FilterGroupComponent = (props: PropsWithChildren<FilterGroupProps>) => {
       return;
     }
     try {
+      if (!Array.isArray(param)) {
+        throw Error('Url Params must be an array');
```

Comment:
hey that one will not include stack trace? im not sure about that

Reply:
correction: looks like it does not make any difference, at least in chrome 
![image](https://user-images.githubusercontent.com/11671118/233669279-0b4bdcae-00d1-4d92-926d-9fb927aefee3.png)

Reply:
@lgestc , yes it was intentional and when I am running the test, it is giving me strack trace.

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/7485038/233671015-5987bbf9-2fdf-4030-b697-dc5e8be4959d.png">

Reply:
@lgestc , also moved the error message to translations.